tests/helpers.py

```diff
@@ -21,9 +21,8 @@
 
 from flask import jsonify
 from flask_login import _create_identifier
-from mock import MagicMock
-
 from invenio_oauthclient.views.client import serializer
+from mock import MagicMock
```

Comment:
It depends how did you install the package dependencies.

For more details please see http://isort.readthedocs.io/en/latest/#configuring-isort